package.json

```diff
@@ -7,9 +7,12 @@
   },
   "scripts": {
     "design:dev": "NODE_ENV=development npm run start:magazine",
-    "design:dev:mocked": "NODE_ENV=development run-p start:json-server start:magazine:watch",
+    "design:dev:mocked": "NODE_ENV=development npm run start:magazine:watch",
     "design:build": "NODE_ENV=production webpack --display-error-details",
     "design:publish": "npm run design:build && ldm design:publish design/dist",
+    "now-build": "npm run design:build",
+    "now-deploy": "now",
+    "now-start": "ENVIRONMENT=now node app/index.js",
     "start:json-server": "nodemon --watch ./mocks --exec 'json-server ./mocks/db.js --routes ./mocks/routes.json --port 3001'",
```

Comment:
Do we still need that and `start:magazine:mocked`?

Reply:
No we don't. Will change now.